tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/inactive/inline_images.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+set -eu
+
+# Replaces %*.png% in the input .html file with a data URI containing the
+# image content of the corresopnding .png files in the images/ subdirectory.
+
+mime_type='image/png'
+
+tmp_file="$(mktemp)"
+cp "$@" "${tmp_file}"
+
+for image_file in $0.runfiles/org_tensorflow_tensorboard/tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/inactive/images/*.png; do
```

Comment:
I don't think this path will be compatible in google3. Please just pass the whole image path as the input (srcs)?